packages/features/webhooks/components/WebhookForm.tsx

```diff
@@ -233,6 +233,7 @@ const WebhookForm = (props: {
                     { value: "default", label: t("default") },
                     { value: "custom", label: t("custom") },
                   ]}
+                  isFullWidth={true}
```

Comment:
design issue in mobile view
![Screenshot 2023-01-31 at 10 37 38](https://user-images.githubusercontent.com/30310907/215805632-d92327f9-8764-4c99-a6b0-740f8fbabeb1.png)

Reply:
![image](https://user-images.githubusercontent.com/69904519/215810604-b7cdae4a-4343-4e20-a6fb-37bf8a3a267d.png)

It's working on my end. Do you know what the issue will be?

Reply:
And what about the ping test button? Should we keep it like that?

Reply:
> It's ok on my end. Do you know what the issue will be?

Try a bigger screen (Ipad), then you should be able to see the issue 

Also this doesn't look right in mobile for me (looks fine on your screenshot tho, did you change anything after?):
<img width="293" alt="Screenshot 2023-01-31 at 17 32 45" src="https://user-images.githubusercontent.com/30310907/215898753-617b9c0f-eaa6-495c-8c3a-cecf252f0443.png">

Reply:
> And what about the ping test button? Should we keep it like that?

Yes, let's keep it like that

Reply:
> > It's ok on my end. Do you know what the issue will be?
> 
> Try a bigger screen (Ipad), then you should be able to see the issue
> 
> Also this doesn't look right in mobile for me (looks fine on your screenshot tho, did you change anything after?): <img alt="Screenshot 2023-01-31 at 17 32 45" width="293" src="https://user-images.githubusercontent.com/30310907/215898753-617b9c0f-eaa6-495c-8c3a-cecf252f0443.png">

No I didn't change anything.